src/lib/components/icons/GluonLogoGreen.tsx

```
@@ -0,0 +1,59 @@
+import React from "react";
+import { tokenConfig } from "@/config/tokenConfig";
+
+export const GluonLogoGreen = ({ className }: { className?: string }) => {
+  const logoColor = tokenConfig.theme.primary;
+
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 214.73 223.66" className={className}>
```

Biome (2.4.6)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)